modules/twinklespeedy.js

```diff
@@ -248,6 +251,21 @@ Twinkle.speedy.initDialog = function twinklespeedyInitDialog(callbackfunc) {
 			}
 		]
 	});
+	tagOptions.append({
+		type: 'checkbox',
+		list: [
+			{
+				label: 'Request creation protection',
+				value: 'salting',
+				name: 'salting',
+				tooltip: 'When selected, tagging the page for deletion will be accompanied by a request for the deleting administrator to apply creation protection.',
```

Comment:
I think it'd be better to make this a little more forceful and clear *when* it should be used.  That is, something like "only use if the page or title has been repeatedly been created" but, you know, better.  As above, this could also be a little more clear as to what will happen (i.e. appending {{salt}}).

Reply:
Done